### PR TITLE
[13] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by elizabethbusola**
## Task: [13] Style the Product Details Page using CSS
- I styled the product page using css and named the file, index.css
Mobile_device
Tablet
webpage

### CODE REVIEW

**Comment by elizabethbusola** [comment]
> @elizabethbusola Well done! You just need to remove the extra spaces in the file

@ToluAkin I have removed the extra spaces...

File: submissions/busolaAdedoyin/css/index.css [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+   .img-lower {
+      /* margin-right: 100px; */
+      float:none;
+      width: 280px;
+      margin-left:0px;
+   }
+
+}
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:259]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+   margin: 0;
+
+}
+
+/*MEDIA QUERY */
+
+@media screen and (max-width: 950px) {   
+ .img-lower {
+    width: 280px;
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:214]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+   height:160px;
+   margin: 15px 85px 0 85px;
+}
+
+.img-lower p {
+   text-align: center;
+   font-weight: bold;
+   padding: 0;
+   margin: 0;
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:206]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+}
+
+.img-lower {
+   float: left;
+   width: 30%;
+   height: 250px;
+   margin:20px;
+   background-color: whitesmoke;
+   margin-bottom: 20px;
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:192]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+   padding: 8px;
+   margin-top: 10px;
+}
+.word {
+   /* float: left; */
+   margin: auto;
+   padding-left: 20px;
+}
+/*LOWER CONTAINER */
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:170]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+   color: white;
+   text-decoration: none;
+}
+
+footer {
+   background-color:#333366;
+   overflow: hidden;
+   padding: 20px 0 0 20px;
+   border-top:4px solid yellow;
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:62]: Extra space here

File: submissions/busolaAdedoyin/css/index.css [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+}
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/css/index.css:64]: Extra space here

File: submissions/busolaAdedoyin/introductionToHtml/index.html [hunk 8]
```diff
@@ -1,95 +1,104 @@
...
+                </div>
+                <div class="img-lower img3">
+                    <img src="img/polishfab.jpg">
+                    <p>Polish Fabric</p>
+                    <p>Price:#8,750</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/introductionToHtml/index.html:73]: Remove extra space here @elizabethbusola

**Comment by elizabethbusola** [comment]
> @elizabethbusola the images in the logos folder, where are you using them?

@ToluAkin I am using them inside htmlForms Folder, I used them for the Contact Page and the signUp Page

**Comment by ToluAkin** [comment]
> > @elizabethbusola the images in the logos folder, where are you using them?
> 
> @ToluAkin I am using them inside htmlForms Folder, I used them for the Contact Page and the signUp Page

Seen. There is conflict on your pull request. @elizabethbusola

**Pull request merged**